### PR TITLE
Added a new page for log events. Created a spot for it in the user gu…

### LINKED ISSUE #615: Add a new page for the WebHooks log event plugin.
There are pages for the other three plugins under WebHooks handlers (run job, advanced run job and PagerDuty). Need to add a page for WebHook log event plugin. I would assume a new page under handlers like the three other pages for the other WebHook plugins. To be confirmed by @fdevans. 

These are the pages that we already have for the other WebHooks plugins:

https://docs.rundeck.com/docs/manual/webhooks/run-job.html#usage
https://docs.rundeck.com/docs/manual/webhooks/advanced-run-job.html#general-settings
https://docs.rundeck.com/docs/manual/webhooks/pagerduty-run-job.html

### PR DESCRIPTION
**Pull request by chrismcg14**
…ide on the side. Added links to the webhooks index page. fixes #615

### CODE REVIEW

File: docs/.vuepress/sidebar-menus/user-guide.js [hunk 1]
```diff
@@ -107,6 +107,7 @@ module.exports = [{
           sidebarDepth: 2,
           children: [
             '/manual/webhooks/run-job.md',
+            '/manual/webhooks/log-events.md',
```
Comment (fdevans) [inline docs/.vuepress/sidebar-menus/user-guide.js]: Let's adjust these so they are "Enterprise First" :)

File: docs/manual/12-webhooks.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ Click the `Add` button to add a new webhook.
 
 Fill out the information in the form and select a Webhook event handler:
 * [Run Job](./webhooks/run-job.md)
+* [Log Events](./webhooks/log-events.md)
+* [PagerDuty Run Job (Enterprise)](./webhooks/pagerduty-run-job.md)
```
Comment (fdevans) [inline docs/manual/12-webhooks.md]: Also... Enterprise First

File: docs/manual/webhooks/log-events.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Log Events
+
+All Rundeck users have access to the Log Events Webhook plugin. The log events plugin allows users to log incoming webhook events to the log4j logger rundeck.webhooks.log file. Users can post any data they might want to the webhook and it will be logged. Because of this, it can act as a debug endpoint. For example, maybe you don't exactly know what data the source will be sending. Using the log events plugin, you could log it and analyze it before using a different plugin to handle it.
```
Comment (fdevans) [inline docs/manual/webhooks/log-events.md]: Log4j2 or log4j?  not sure if it needs to be distinct in this instance. @sjrd218

Comment (sjrd218) [reply docs/manual/webhooks/log-events.md]: It would probably be good to change it to log4j2.

**Pull request merged**